### PR TITLE
Validate the firmware when a uart proxy is synced.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: src/jaguar.toit [hunk 1]
```diff
@@ -20,7 +20,7 @@ import .network
 import .uart
 
 interface Endpoint:
-  run device/Device -> none
+  run device/Device [--validate-firmware] -> none
```
Comment (kasperl) [inline src/jaguar.toit:23]: Why is this done through a callback? Why not just import the jaguar.toit file and call validate-firmware? What am I missing?

Comment (floitsch) [reply src/jaguar.toit]: I hoped I would forget less easily if given as argument.
Changed to a static call.
done.

File: src/jaguar.toit [hunk 2]
```diff
@@ -136,6 +138,14 @@ serve arguments:
   else:
     Task.group lambdas
 
+validate-firmware:
```
Comment (kasperl) [inline src/jaguar.toit:141]:
```suggestion
validate-firmware -> none:
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/jaguar.toit (hunk 2 above)
Comment (floitsch) [reply src/jaguar.toit:141]: This has been done in a later PR.

**Pull request merged**